sdcm/cluster_docker.py

```diff
@@ -280,21 +280,14 @@ def __init__(self,
                          base_logdir=base_logdir,
                          ssh_login_info=ssh_login_info,
                          node_index=node_index)
-        self._vector_store_client = None
 
     def node_container_run_args(self, seed_ip=None):
         return self.vector_container_run_args(seed_ip)
 
     def vector_container_run_args(self, seed_ip=None):
-        scylla_uri = "127.0.0.1:9042"
-        if self.parent_cluster.scylla_cluster:
-            scylla_uri = ",".join(
-                f"{node.ip_address}:{self.parent_cluster.params.get('vs_scylla_port')}"
-                for node in self.parent_cluster.scylla_cluster.nodes)
-
         environment = {
             'VECTOR_STORE_URI': f"0.0.0.0:{self.parent_cluster.params.get('vs_port')}",
-            'VECTOR_STORE_SCYLLADB_URI': scylla_uri,
+            'VECTOR_STORE_SCYLLADB_URI': self.scylla_uri,
```

Comment:
need to add follow up task: when decomissioning scylla node we need to update this address (the one that was specified here)